page-members-only.php

```diff
@@ -0,0 +1,72 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+
+<?php
+if( have_posts() ) {
+  while( have_posts() ) {
+    the_post();
+  ?>
+  <article id="page" <?php post_class('container'); ?>>
+    <div class="grid-row margin-bottom-basic">
+      <?php render_divider(get_the_title()); ?>
+    </div>
+    <div class="grid-row">
+      <div class="grid-item item-s-12 item-m-8 offset-m-2 margin-bottom-basic">
+        <div class="post-copy">
+          <?php the_content(); ?>
+        </div>
+      </div>
+    </div>
+
+    <?php
+      $member_posts = new WP_Query(array(
+        'post_type' => array('event', 'recording'),
+        'posts_per_page' => -1,
+/*         'category__in' => 451, */
+        'category__in' => 51,
```

Comment:
**just saying, idc**
theres better ways to do this. the category id should come from slug not an id that could change on a migration or doesnt work on local env, etc